src/quantuminspire/api.py

```diff
@@ -68,6 +72,8 @@ def __init__(self, base_uri: str, authentication: coreapi.auth.BasicAuthenticati
         Raises:
             ApiError: an ApiError exception is raised when the schema could not be loaded.
         """
+        if authentication is None:
+            authentication = TokenAuthentication(load_token(), scheme="token")
```

Comment:
When no token is loaded, the ErrorMessage of coreapi.exceptions says:
<Error: 401 Unauthorized> detail: "Invalid token."

I agree this is misleading because there is no token at all.